src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SWNativeAlignerWrapper.java

```diff
@@ -25,6 +25,24 @@ public SWNativeAlignerWrapper(final SWAlignerNativeBinding aligner) {
         this.aligner = aligner;
     }
 
+
+    //methods for printing out number of SW/non-SW alignments. not needed
+    public int getNumOfAlignments()
```

Comment:
I think these escaped the purge from a few commits ago, but I'm glad they did because it's a good example of Java annotations: since these implemented an abstract class they should have gotten the `@Override` annotation, in which case once the parent class methods were deleted the IDE would have complained about these.